src/components/order-service/order-service.component.ts

```diff
@@ -2,7 +2,8 @@ import {OrderServiceController} from './order-service.controller';
 
 export const orderService: angular.IComponentOptions = {
   bindings: {
-    service: '<'
+    serviceClass: '<',
```

Comment:
No, builder images don't have parameters.

I don't think anything in the current order service controller will work with builder images. Even the metadata is different from service classes.